Formula/filebeat.rb

```diff
@@ -0,0 +1,76 @@
+class Filebeat < Formula
+  desc "File harvester, used to fetch logs files and feed them into logstash"
+  homepage "https://www.elastic.co/products/beats/filebeat"
+  url "https://github.com/elastic/beats/archive/v1.2.3.tar.gz"
+  sha256 "8eea85de415898c362144ba533062651d8891241c738799e54cc9b17040c1fc9"
+
+  head "https://github.com/elastic/beats.git"
+
+  depends_on "go" => :build
+
+  def install
+    gopath = buildpath/"gopath"
+    (gopath/"src/github.com/elastic/beats").install Dir["{*,.git,.gitignore}"]
+
+    ENV["GOPATH"] = gopath
+
+    cd gopath/"src/github.com/elastic/beats/filebeat" do
+      system "make"
+      libexec.install "filebeat"
+      etc.install "etc/filebeat.yml"
+    end
+
+    (bin/"filebeat").write <<-EOS.undent
+      #!/bin/bash
+      #{libexec}/filebeat -c #{etc}/filebeat.yml "$@"
```

Comment:
Let's modify the wrapper to look as follows instead (the `exec` should fix the test problem):

```sh
#!/bin/sh
exec "#{libexec}/filebeat" -c "#{etc}/filebeat.yml" "$@"
```

The `exec` will make sure to replace the wrapper shell process with the `filebeat` process (there's no need to run it as a child process of the wrapper), quoting file names makes sure to avoid issues with the file names (though Homebrew is unlikely to be installed in a location with problematic file paths, e.g. including whitespace), and `/bin/sh` is sufficient (and more portable) as this isn't using any Bash-specific features.